spec/requests/api/v1/targets/delete_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
+
+describe 'DELETE api/v1/targets/:id', type: :request do
+  let(:user_with_targets) { create(:user_with_targets) }
```

Comment:
Just a suggestion here, I would name this variable just `user` to make it more readable when you call `user.targets` for example